doc/nrf/gs_programming.rst

```diff
@@ -86,6 +87,53 @@ You can download it from the following links:
    .. note::
    	In a multi-image build, this allows you to debug the source code of your application only.
 
+
+.. _gs_programming_manage:
+
+Managing projects with SES
+=========================
+
+Using SES it is possible to add/remove files to the application project using the SES.
```

Comment:
The final "using the SES" is redundant and can be removed.

Reply:
Thanks, will check up, when @ru-fu makes her doc updates